Tests/UnitTests/Networking/HTTPStatusCodeTests.swift

```diff
@@ -90,7 +92,6 @@ class HTTPStatusCodeTests: TestCase {
         expect(HTTPStatusCode.redirect.isSuccessfullySynced) == true
         expect(HTTPStatusCode.notModified.isSuccessfullySynced) == true
         expect(HTTPStatusCode.temporaryRedirect.isSuccessfullySynced) == true
-        expect(HTTPStatusCode.invalidRequest.isSuccessfullySynced) == true
```

Comment:
This was also wrong?

Reply:
I wonder if we were doing this so that if there's bad subscriber attribute data was just mark is as synced to avoid posting it forever?
If that's the case, we should change the implementation to allow `400` but not all `4xx`.